tableaudocumentapi/archivefile.py

```diff
@@ -0,0 +1,67 @@
+import contextlib
```

Comment:
containerfile works for me.

<Insert joke about naming things is hard here / >